tests/oauth.rs

```diff
@@ -276,6 +312,26 @@ async fn normal_flow() {
 
 		assert!(data["id"].is_number());
 		assert_eq!(data["username"], user_username);
+
+		http_client.post("/logout").dispatch().await;
+
+		let form_body = format!("token=1234");
```

Comment:
```suggestion
		let form_body = format!("token={}", token);
```
I think you meant to test if the token is invalidated?

Reply:
yes and the test fails :cry: